ansible_collections/arista/avd/tests/unit/filters/test_range_expand.py

```diff
@@ -25,6 +25,24 @@
     "65100.0-4",
     "65100.0-2,65200.1-2",
     "1-2.0-1",
+    "Ethernet(1)",
+    "Ethernet(1,2,3)",
+    "Ethernet(1)/(4,5)",
+    "(1,2)",
+    "1,(2,3)/5",
+    "Eth(10-14)",
+    "Eth(101-103)/(5,7)",
+    "Eth(1,2",
+    "Eth(1,2)/3,4)",
+    "Eth(1,2)/3,4",
+    "Eth(4)/(2-3)/(1,32)/4.(200-202,300)",
+    ["(1-5)", "(2,3,4)", "14-16"],
+    ["Ethernet 1-2/(3-5)", "Eth3-5/(3,4,5)", "7-8/(3,4,5)"],
+    ["Et (3,4,5).1", "Et4/(3,4,5).1", "Et5.200", "Et(4-6).200"],
+    "(65000).1",
+    ["Pt(600, 7000)/2.1", "Pt( 600, 7000)/2.1", "Pt(600, 7000 )/2.1"],
+    "eth1-5,(7,9,11-13)",
+    "eth(7,9,11-13),21,26",
 ]
 
 EXPECTED_RESULT_VALID_VALUES = [
```

Comment:
Can you refactor the constants here to be a list of tuples, where the value is the first element and the expected result is the second? That will make it easier to pair them up and ensure we have the logic we want.
Thanks

Reply:
Done :)